scripts/android-build-shared-library.sh

```diff
@@ -0,0 +1,49 @@
+NODEJS_HEADERS_DIR="$( cd ../../ && cd nodedir/android/libnode/ && pwd )"
+
+ARCH=$1
+
+# Build for arm64 by default
+if [ -z $ARCH ]
+then
+  ARCH="arm64"
+fi
+
+# Choose proper target basing on architecture we build against
+if [ $ARCH == "arm64" ]
+then
+  TARGET="aarch64-linux-android"
+fi
```

Comment:
Maybe add else, echo message that wtf and exit 1

Reply:
Good point